src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataResource.java

```diff
@@ -105,6 +110,14 @@ public ResponseEntity<GetCaseSummaryResponse> getCaseSummary(@PathVariable UUID
         return ResponseEntity.ok(GetCaseSummaryResponse.from(caseSummary));
     }
 
+    @GetMapping(value = "/case/{caseUUID}/team/members")
+    public ResponseEntity<List<UserDto>> getCaseTeams(@PathVariable UUID caseUUID) {
+        Set<UUID> teamUUIDs = caseDataService.getCaseTeams(caseUUID);
+        List<UserDto> users = teamUUIDs.stream().map(infoClient::getUsersForTeam).flatMap(List::stream).collect(Collectors.toList());
```

Comment:
It can be joined into a single call to info but as far as I can tell we would still require multiple calls to keycloak from info as I don't believe it is possible to query by multiple teams in keycloak.

Would this still be worth doing regardless to reduce the number of info calls?

Reply:
As this is an existing method and already used - while it is non-performant should be looked at from a wider piece with Keycloak.